### PR TITLE
chore: enable ct testing on console

### PR DESCRIPTION
**Pull request by alejandroEsc**
This allows for a workflow for console, we turn off tls for now as we need to come up with a working solution for this case.

### CODE REVIEW

File: .github/workflows/pull_requests_console.yaml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        run: |
+          kubectl -n metallb-system apply -f .github/metallb-config.yaml
+
+        # Would like to enable tls but this requires additional work
+      - name: install redpanda
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add redpanda https://charts.redpanda.com &&
+          helm install redpanda redpanda/redpanda --namespace redpanda  \
+            --set tls.enabled=false \
```
Comment (RafalKorepta) [inline .github/workflows/pull_requests_console.yaml:156]: Good Job.

As a follow up I would suggest to create another Redpanda listener that has TLS enabled.

File: .github/workflows/pull_requests_console.yaml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            --github-groups \
+            --upgrade \
+            --config .github/ct-console.yaml \
+            --helm-extra-set-args=" \
+          --set=console.config.kafka.tls.enabled=false \
+          --set=console.config.kafka.brokers={redpanda-0.redpanda.redpanda.svc.cluster.local.:9093,redpanda-1.redpanda.redpanda.svc.cluster.local.:9093,redpanda-2.redpanda.redpanda.svc.cluster.local.:9093} \
+          --set=console.config.kafka.sasl.enabled=false \
+          --set=console.config.kafka.schemaRegistry.enabled=true \
+          --set=console.config.kafka.schemaRegistry.tls.enabled=false \
+          --set=console.config.kafka.schemaRegistry.urls={http://redpanda-0.redpanda.redpanda.svc.cluster.local.:8081,http://redpanda-1.redpanda.redpanda.svc.cluster.local.:8081,http://redpanda-2.redpanda.redpanda.svc.cluster.local.:8081}" \
```
Comment (RafalKorepta) [inline .github/workflows/pull_requests_console.yaml:172]: Why those configs aren't put into ci folder?

File: .github/workflows/pull_requests_console.yaml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add metallb https://metallb.github.io/metallb &&
+          helm install metallb metallb/metallb -n metallb-system \
+            --create-namespace --version 0.13.10 --wait --wait-for-jobs
+
+      - name: apply metallb resources
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          kubectl -n metallb-system apply -f .github/metallb-config.yaml
```
Comment (RafalKorepta) [inline .github/workflows/pull_requests_console.yaml:148]: Why metallb is added?

File: .github/workflows/pull_requests_console.yaml [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+      - name: install prometheus-operator
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add prometheus-community https://prometheus-community.github.io/helm-charts &&
+          helm install prometheus prometheus-community/kube-prometheus-stack \
+            --namespace prometheus --create-namespace \
+            --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false \
+            --set nodeExporter.enabled=false \
+            --set grafana.enabled=false --set kubeStateMetrics.enabled=false --set alertmanager.enabled=false \
+            --wait --wait-for-jobs
```
Comment (RafalKorepta) [inline .github/workflows/pull_requests_console.yaml:136]: Why Prometheus operator is added?

File: .github/workflows/pull_requests_console.yaml [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+        with:
+          config: .github/kind.yaml
+
+      - name: Check kind config worked
+        if: steps.list-changed.outputs.changed == 'true'
+        run: kubectl get nodes
+
+      - name: Annotate nodes for rack awareness
+        if: steps.list-changed.outputs.changed == 'true'
+        run: .github/annotate_kind_nodes.sh chart-testing
```
Comment (RafalKorepta) [inline .github/workflows/pull_requests_console.yaml:117]: Why rack awareness script is used?

**Review by RafalKorepta — APPROVED** [review]
I see there are too many steps, but other than that LGTM

**Comment by alejandroEsc** [comment]
> I see there are too many steps, but other than that LGTM

yep, you made some good suggestions, ill simplify

**Pull request closed without merging**